### PR TITLE
Attempted fix to #39

### LINKED ISSUE #39: Raw translation string appearing in ore customization screen
Pretty low priority, but cubicgen.gui.cubicgen.select_biomes doesn't seem like something that should be appearing.

### LINKED ISSUE #91: Add a warning to CustomCubic settings about some structures not being supported yet
Currently mineshafts, villages, temples and plenty other vanilla structures aren't supported in CustomCubic, but options to toggle them on and off are. Having these options turned on, only to not have them generate in the world anyway, seems potentially a bit confusing, so something to clarify why they don't spawn would probably be helpful.

### PR DESCRIPTION
**Pull request by muzikbike**
closes #39, closes #91

### CODE REVIEW

File: src/main/resources/assets/cubicgen/lang/en_us.lang [hunk 1]
```diff
@@ -42,21 +42,43 @@ cubicgen.gui.cubicgen.from=From:
...
 cubicgen.gui.cubicgen.dungeonCount=Dungeon Count
-cubicgen.gui.cubicgen.lavaOceans=Lava Oceans
-cubicgen.gui.cubicgen.biome=Biome
-cubicgen.gui.cubicgen.biomeSize=Biome Size
-cubicgen.gui.cubicgen.riverSize=River Size
+
+cubicgen.gui.cubicgen.strongholds=Strongholds
+cubicgen.gui.cubicgen.strongholdDistance=Stronghold Distance*
+cubicgen.gui.cubicgen.strongholdCount=Stronghold Count*
+cubicgen.gui.cubicgen.strongholdSpread=Stronghold Spread*
```
Comment (Niko-sk2x) [inline src/main/resources/assets/cubicgen/lang/en_us.lang]: Don't add unused lang entries. They will get added when they are needed.

**Comment by muzikbike** [comment]
removed

**Comment by Niko-sk2x** [comment]
Did you actually test it? Also avoid changing order... makes the diff too hard to follow.

**Comment by muzikbike** [comment]
just tested it, seems to be functioning properly

File: src/main/resources/assets/cubicgen/lang/en_us.lang [hunk 2]
```diff
@@ -65,6 +74,8 @@ cubicgen.gui.cubicgen.riverSize=River Size
 cubicgen.gui.cubicgen.add_ore=Add ore
 cubicgen.gui.cubicgen.delete=Delete
 
+cubicgen.gui.cubicgen.select_biomes=Restrict to biomes...
```
Comment (Niko-sk2x) [inline src/main/resources/assets/cubicgen/lang/en_us.lang:77]: I don't remember that in GUI

File: src/main/resources/assets/cubicgen/lang/en_us.lang [hunk 3]
```diff
@@ -42,21 +42,30 @@ cubicgen.gui.cubicgen.from=From:
...
+
+cubicgen.gui.cubicgen.strongholds=Strongholds
+
+cubicgen.gui.cubicgen.villages=Villages*
+
+cubicgen.gui.cubicgen.temples=Temples*
+
+cubicgen.gui.cubicgen.oceanMonuments=Ocean Monuments*
+
+cubicgen.gui.cubicgen.woodlandMansions=Woodland Mansions*
```
Comment (Niko-sk2x) [inline src/main/resources/assets/cubicgen/lang/en_us.lang:68]: Why is it moved?

File: src/main/resources/assets/cubicgen/lang/en_us.lang (hunk 2 above)
Comment (muzikbike) [reply src/main/resources/assets/cubicgen/lang/en_us.lang:77]: this is #39

File: src/main/resources/assets/cubicgen/lang/en_us.lang (hunk 3 above)
Comment (muzikbike) [reply src/main/resources/assets/cubicgen/lang/en_us.lang:68]: to have structures grouped together, should this be reverted?
